src/index.html

```diff
@@ -1,13 +1,366 @@
+              <a href="#" class="button" target="_blank">View all</a>
+            </div>
+          </div>
+
+
+
+        </div>
+      </section>
+
+      <section class="page__section testemonials" id="testimonials">
+        <div class="container">
+          <!-- <h2 class="page__section-t">Why buy direct from Bose</h2> -->
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-5">
+              <h2 class="testemonials__title">Why buy direct from Bose</h2>
+            </div>
+
+            <div class="grid__item grid__item--6-10">
+              <h3 class="testemonials__subtitle">Free 2-day shipping and returns</h3>
+              <h3 class="testemonials__subtitle">90-day risk-free trial</h3>
+              <h3 class="testemonials__subtitle">World class customer service</h3>
+              <h3 class="testemonials__subtitle">My Bose account management</h3>
+              <p class="testemonials__text">A great product is more than what’s in the box.
+                It’s a promise of premium performance, world-class support,
+                and everything you expect from a trusted brand. It’s just
+                one of many reasons why you’ll shop with confidence on Bose.com.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contacts" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title">Contact us</h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-5">
+              <!-- <div class="contacts__form"></div> -->
+              <form action="#" class="contacts__form">
+                <label for="name">
+                  <input
+                    id="name"
+                    name="name"
+                    type="text"
+                    placeholder="Name"
+                    class="contacts__field form-field"
+                    required
+                  >
+                </label>
+                <label for="email">
+                  <input
+                    id="email"
+                    name="email"
+                    type="email"
+                    placeholder="Email"
+                    class="contacts__field form-field"
+                    required
+                  >
+                </label>
+                <!-- <input
+                  id="email"
+                  name="email"
+                  type="email"
+                  placeholder="Email"
+                  class="contacts__field form-field"
+                > -->
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  class="
+                    contacts__field
+                    contacts__field--last
+                    form-field
+                    form-field--textarea"
+                    required
+                >
+                </textarea>
```

Comment:
```suggestion
               ></textarea>
```
Зараз не відображається плейсходер в trxtarea. І текст вводиться некоректно. Існує такий баг. Він фікситься таким форматуванням, як я показав - ОДИН відступ вліво для закриваючої дужки і закриваючий тег на тому ж рядку, що і дужка

<img width="487" alt="Снимок экрана 2022-03-03 в 13 29 23" src="https://user-images.githubusercontent.com/66570142/156556935-b1ddd7a1-0774-4c5c-90be-4d917f8a50a5.png">